src/Nest/Document/Multiple/BulkAll/BulkAllRequest.cs

```diff
@@ -47,7 +47,11 @@ public interface IBulkAllRequest<T> where T : class
 		/// </summary>
 		Action<BulkResponseItemBase, T> DroppedDocumentCallback { get; set; }
 
-		///<summary>Default index for items which don't provide one</summary>
+		///<summary>The index to use for items that don't specify one. By default, will be inferred from <typeparamref name="T"/>.
```

Comment:
```suggestion
		///<summary>The index to use for documents that don't specify one. By default, will be inferred from <typeparamref name="T"/>.
```

Reply:
I originally wrote documents :) I changed it to items because the `IBulkAllRequest<T>` sends bulk request items which might include documents in the case of index operations, but might not for others like delete operations, although bulking deletes would be an unusual usage of `BulkAll`. The other thought was that _documents don't specify the index_, but maybe we can come up with better wording for this?

Reply:
Ah, yes, I see the problem now :) In that case I think "items" is fine as a cover-all term.

If we really wanted to be pedantic perhaps "documents and bulk operations" - but I think this starts to become confusing too!

Reply:
I'll leave as is for the moment. Thanks for looking 👍